src/schema/schema.h

```diff
@@ -119,3 +134,43 @@ void Schema_Free
 (
 	Schema *s
 );
+
+bool Schema_HasConstraints(const Schema *s);
+
+// retrieves constraint 
+// returns NULL if constraint was not found
+Constraint Schema_GetConstraint
```

Comment:
I think we need a function that will get all constraints 
fulfilling the following
input_field_list ⊆ constraint fields

```c
Constraint *Schema_GetConstraintsSubSet
(
 	const Schema *s,         // schema from which to get constraint
 	const AttrInfo *fields,  // constraint fields as a subset
 	uint field_count         // number of fields
)
```
For example
`fields = [1,2]`
And there are two constraints `A` with the following fields: [1,2,3] and `B` with the following fields: [1,3]
`A` should be part of the returned array of constraints while `B` shouldn't.